x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferenceNamedWriteablesProvider.java

```diff
@@ -96,8 +96,8 @@ public static List<NamedWriteableRegistry.Entry> getNamedWriteables() {
         namedWriteables.add(
             new NamedWriteableRegistry.Entry(
                 ServiceSettings.class,
-                TextEmbeddingInternalServiceSettings.NAME,
-                TextEmbeddingInternalServiceSettings::new
+                ElasticsearchInternalServiceSettings.NAME,
+                ElasticsearchInternalServiceSettings::new
```

Comment:
do we still need "Internal" to be a part of this class name?

Reply:
Not really. I don't have a strong opinion, my motivation was I thought it would be easier to find the file in an IDE using Go To File as there are a lot of classes that start with Elasticsearch